test/aws/mock_data.py

```diff
@@ -20,6 +21,34 @@
 TEST_CLOUDFRONT_ID = "test-cloudfrontet-TEST"
 TEST_CLOUDFRONT_DOMAIN_NAME = "test-s3.cloudfront.net"
 INSTANCE_ID = "test-instance"
+VM_BUCKET_OUTPUTS = {
+    "VMBucketId": TEST_BUCKET_ID,
+    "VMExportRoleId": TEST_ROLE_ID,
+    "CfDistributionId": TEST_CLOUDFRONT_ID,
+    "CfDistributionDomainName": TEST_CLOUDFRONT_DOMAIN_NAME,
+}
+EXAMPLE_DATA_BUCKET_OUTPUTS = {
+    "ExampleDataBucketId": TEST_BUCKET_ID,
+    "ExportRoleId": TEST_ROLE_ID,
+    "CfDistributionId": TEST_CLOUDFRONT_ID,
+    "CfDistributionDomainName": TEST_CLOUDFRONT_DOMAIN_NAME,
+}
+VM_BUCKET_WAF_OUTPUTS = { 'VMDownloadACLArn': TEST_ACL_ARN }
+EXAMPLE_DATA_WAF_OUTPUTS = { 'AiLabExampleDataDownloadACLArn': TEST_ACL_ARN }
+
+
+def cf_stack_outputs(class_name: str, category: str) -> Dict[str, any]:
+    outputs = {
+        "VmBucketCfTemplate": {
```

Comment:
Is this string ("VmBucketCfTemplate") defined anywhere in the code? If yes, use a constant

Reply:
I replaced this function by regular `@pytest.mark.parametrize`.
See next push.